_sources/developer/github.rst.txt

```diff
@@ -0,0 +1,99 @@
+******************
+Github integration
+******************
+
+There are several files in the repository that serve to manage the interaction
+between GitHub and the repository, and other settings in GitHub that are
+relevant. Generally administrators manage thes settings, but anybody can
+submit a pull request that changes the files managing this interaction.
+
+.. contents::
+   :local:
+
+Documentation
+=============
+Several documentation files are also rendered by Github. See
+:ref:`documentation-magic-github`.
+
+Issue and Pull Request Templates
+================================
+These files are in the ``.github`` directory of the repository, and documented
+`at GitHub <https://docs.github.com/en/communities/
+using-templates-to-encourage-useful-issues-and-pull-requests/
+about-issue-and-pull-request-templates>`_.
+
+Settings
+========
+See `repository roles <https://docs.github.com/en/organizations/
+managing-access-to-your-organizations-repositories/
+repository-roles-for-an-organization>`_.
+     
+Website
+=======
+This is stored on a separate branch; `this page <https://docs.github.com/
+en/pages/getting-started-with-github-pages/
+configuring-a-publishing-source-for-your-github-pages-site>`_ describes
+how it is rendered.
+
+CircleCI
+========
+Continuous integration is managed through `CircleCI <https://circleci.com/>`_.
+
+Some project settings:
+
+"Only build pull requests" is on; we will not build all branches.
+
+CircleCI refers to a PR that is relative to a fork as a "forked PR". Because
+this the preferred means of submitting a PR, "Build forked pull requests"
+is on. (The PR itself must actually be submitted to the dbprocessing
+repository).
+
+We require status checks to pass before merging to master; in the GitHub
+branch settings, this takes the form of a branch protection rule that requires
+the CircleCI ``main`` workflow to succeed on the PR. `This document
+<https://web.archive.org/web/20210304061613/
```

Comment:
I checked; that's the correct link. The full sentence is "[This document](https://web.archive.org/web/20210304061613/https://support.circleci.com/hc/en-us/articles/360004346254-Workflow-status-checks-never-completes-because-of-ci-circleci-Waiting-for-status-to-be-reported) appears to state the top-level circleci check should not be selected" and the contents of the page are "you'll need to make sure the ci/circleci check is unchecked..." The problem is CircleCI removed that information from their website so I had to link to a cached copy.